src/acktest/bootstrapping/dynamodb.py

```diff
@@ -29,7 +29,7 @@ def dynamodb_resource(self):
     def bootstrap(self):
         """Creates a Dynamodb table with an auto-generated name.
         """
-        self.name = resources.random_suffix_name(self.name_prefix, 63)
+        self.name = resources.random_suffix_name(self.name_prefix, 8)
```

Comment:
You can easily blow past 8 character limit with a table name from tests. Why did you need to change this?

Reply:
Modified the wrong file :) just force pushed

Reply:
Signing profile names cannot go over 64characters

Reply:
why not make it something like 32? instead of 8.

IIRC, this method will only add suffix if name_prefix is less than 8 chars which is very less IMO. Consider 32 if you don't mind

Reply:
Agreed